fiware_api_blueprint_renderer/themes/default_theme/fragments/api_blueprint.tpl

```diff
@@ -186,7 +198,19 @@
                             {% for action in resource.actions %}
                                 {# Display Action if it has at least one example #}
                                 {%- if action["has_example"] -%}
-                                    <div id="{{ action.id }}_examples" class="actionExample {{action.method}}">
+                                    {% if action.method == "GET" %}
+                                        {% set contextual_background = "bg-success" %}
```

Comment:
Call the CSS class as `bg-success` could be confusing. Consider to use another name or duplicate it.
The same  for `bg-info`, `bg-warning` and `bg-danger`